File: UI/UI/src/app/core/model/camera.ts
export class Camera {
  port: number;
  width: number;
  height: number;
  name: string;
  src: string;

  constructor(port: number, name: string, src: string, width: number, height: number) {
    this.port = port;
    this.name = name;
    this.src = src;
    this.width = width;
    this.height = height;
  }
}

File: API/API/Model/Camera.cs
using System.Text.Json;

namespace API.Model
{
    public class Camera :BaseEntity
    {
        public string Name { get; set; } = null!;
        public string Src { get; set; } = null!;
        public int Port { get; set; }
        public int Width { get; set; }
        public int Height { get; set; }


        public override string ToString()
        {
            return JsonSerializer.Serialize(this);
        }
    }
}
